tests/m2m_through_regress/models.py

```diff
@@ -50,7 +50,7 @@ class Meta:
         ordering = ('name',)
 
     def __str__(self):
-        return "%s" % self.name
+        return str(self.name)
```

Comment:
`name` is a `CharField` so `str()` is unnecessary.

Reply:
In this instance it is nullable so we have to call `str()` as the return type of `__str__()` must be `str`.